pymc/backends/arviz.py

```diff
@@ -63,6 +63,43 @@ def find_observations(model: Optional["Model"]) -> Optional[Dict[str, Var]]:
     return observations
 
 
+def find_constants(model: Optional["Model"]) -> Optional[Dict[str, Var]]:
+    """If there are constants available, return them as a dictionary."""
+    # For constant data, we are concerned only with deterministics and
+    # data. The constant data vars must be either pm.Data
+    # (TensorConstant/SharedVariable) or pm.Deterministic
+
+    if model is None:
+        return None
+
+    observations = find_observations(model)
```

Comment:
Similarly if we make this return an empty dict we don't need to worry about the `None` case below.